### PR TITLE
Fix incorrect port

### PR DESCRIPTION
**Pull request by mattppal**
Small update to contributor guide

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
why do we update the port? 3000 is the port for frontend. 6789 is the port for backend.

**Comment by mattppal** [comment]
@wangxiaoyou1993 updated

**Review by wangxiaoyou1993 — APPROVED** [review]

File: docs/contributing/development-environment.mdx [hunk 1]
```diff
@@ -42,7 +42,7 @@ Next, you'll need to configure your development environment. This should be as s
    ```
 </Step>
 <Step title="Open Mage">
-    Open your browser and go to [http://localhost:3000](http://localhost:3000).
+    When developing frontend, use port 3000. If you only need to develop backend, you can use port 6789. Find these ports at [https://localhost:3000](https://localhost:3000) & [https://localhost:6789](https://localhost:6789).
```
Comment (dy46) [inline docs/contributing/development-environment.mdx:45]: for localhost, i believe it should be `http` not `https`

Comment (mattppal) [reply docs/contributing/development-environment.mdx:45]: thx

**Review by dy46 — APPROVED** [review]

**Pull request merged**